idl/partition_checksum.idl.hh

```diff
@@ -110,13 +110,20 @@ enum class node_ops_cmd : uint32_t {
      removenode_sync_data,
      removenode_abort,
      removenode_done,
+     replace_prepare,
+     replace_prepare_mark_alive,
+     replace_prepare_pending_ranges,
+     replace_heartbeat,
+     replace_abort,
+     replace_done,
 };
```

Comment:
Do we need a cluster feature to prevent the new commands being sent to nodes that don't recognize them?

Probably, it's not critical since this is a rare operation, but we need to make sure new nodes respond correctly to responses from old nodes.

Reply:
Yes, old olds are handled below.